api/src/main/java/org/openmrs/aop/AuthorizationAdvice.java

```diff
@@ -44,15 +44,13 @@
 	@Override
 	public void before(Method method, Object[] args, Object target) throws Throwable {
 		
-		if (log.isDebugEnabled()) {
-			log.debug("Calling authorization advice before " + method.getName());
-		}
+		log.debug("Calling authorization advice before {}", method.getName());
 		
 		if (log.isDebugEnabled()) {
```

Comment:
Did you just forget this if statement?

Reply:
I thought it's needed, because it is less expensive to check if debug enabled than do all these things without any sense. Should I also remove it?

Reply:
Well, your use case makes great sense to me.

Reply:
@samuelmale Actually if the `log.debug` is expensive only (if contains any concatenations, or covering to string like that), we usually use the if statement to avoid the unwanted expensive executions during the non-debug instances. 
Other than that, we can simply skip the IF statement since that line is only for printing the simple log.

@dkayiwa  Could you give your feedback on this?